### DISCUSSION TITLE
PPStructure版面分析时对原图返回的是figure，对原图灰度化后却返回table是为什么，请问怎么指定他返回figure？

### DISCUSSION THREAD

**Discussion by jamesbonds01** [PaddlePaddle/PaddleOCR#13125] [Q&A]
#### 问题描述 / Problem Description

版面分析时需要去除原图中的红色像素点，使用cv2.cvtColor(RedThresh,cv2.COLOR_GRAY2BGR)将去除红色像素后的灰度图转换为RGB图之后，PPStructure读取后就只返回table类型的值了，导致后面的我们写的算法无法进行，请问怎么指定他返回figure类型，table类型里面为什么都是html格式的，这种怎么转成figure类型里的json格式？
#### 运行环境 / Runtime Environment
- OS:
- Paddle:
- PaddleOCR:2.7.3

**Comment by GreatV** [comment]
在使用PPStructure进行版面分析时，返回结果可能会受到输入图像的特征和预处理方式的影响。如果对原图进行分析返回的是figure，但对灰度化后的图像分析返回的是table，这可能与模型在处理不同图像特征时的行为有关。

**Reply by jamesbonds01** [reply]
怎么强制让他按figure分析啊，人眼看到的两张图差别不大啊

**Comment by zhangtingyun** [comment]
你好，想请问你的是中文数据集还是英文数据集，正确率大概多少呢？

**Comment by jamesbonds01** [comment]
中文数据集
 崔虎
***@***.***
  

Original:
From:zhangtingyun< ***@***.***>Date:2024-07-01 11:17:35To:PaddlePaddle/PaddleOCR< ***@***.***>Cc:崔虎< ***@***.***>、Author< ***@***.***>Subject:Re: [PaddlePaddle/PaddleOCR] PPStructure版面分析时对原图返回的是figure，对原图灰度化后却返回table是为什么，请问怎么指定他返回figure？ (Discussion #13125)
你好，想请问你的是中文数据集还是英文数据集，正确率大概多少呢？

**Reply by zhangtingyun** [reply]
方便分享一下  训练参数 这样子的吗？  我的模型训练下来准确率一直在 80左右，无法提升
